python/setup.py

```diff
@@ -15,11 +15,14 @@ def get_lua_scripts():
 
     paths = []
 
+    if not os.path.exists(SCRIPTS_PATH):
```

Comment:
Is this a bug fix? I might be missing how this is related to the python3 compat changes.

Reply:
This packaging code was not previously called in CI, and it failed. Now that we use `tox`, we actually run `setup.py` in CI. So it's actually increased code coverage.

Reply:
I think it's a bug fix